docs/academy/moonbeam_course/lesson3.md

```diff
@@ -0,0 +1 @@
+# Lesson 3: How to Index a Substrate Event
```

Comment:
Are you planning on adding these before we merge?

Reply:
Of course. I am working on this.